packages/grid/test/physical-count-polymer.test.js

```diff
@@ -0,0 +1,2 @@
+import '../all-imports.js';
```

Comment:
Same as above, `vaadin-grid.js` import would be enough.

Reply:
Done